.github/workflows/publish_navigation_component_mavencentral.yml

```diff
@@ -0,0 +1,85 @@
+# This workflow will build a Java project with Gradle
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-gradle
+
+name: publish-navigation-component-mavencentral
+# This workflow builds and publishes in Maven Central the Foldable Navigation Component artifacts (all of them).
+# Although the final publishing step has to be done manually in Sonatype site.
+
+on:
+  push:
+    tags:
+      - 'dualscreen*'
```

Comment:
Please change it for 'navigation*' since we will use that key-word to trigger this workflow

Reply:
Done